pkg/csource/generated.go

```diff
@@ -5168,6 +5168,373 @@ static long syz_genetlink_get_family_id(volatile long name)
 }
 #endif
 
+#if SYZ_EXECUTOR || SYZ_VHCI_INJECTION
+/*
+ *
+ *  BlueZ - Bluetooth protocol stack for Linux
+ *
+ *  Copyright (C) 2000-2001  Qualcomm Incorporated
```

Comment:
How much code comes from that source? Please re-type it.
Acquiring some 2000-era Qualcomm owned copyrights is not fascinating.